server/models/Request.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  sitterId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  start: {
+    type: Date,
+    required: true,
+  },
+  end: {
+    type: Date,
+    required: true,
+  },
+  accepted: {
```

Comment:
Additionally, we can consolidate the `accepted/declined` fields under a single enum type object.

Example: 
```
status: {
    type: String,
    enum: ['pending', 'accepted', 'declined', 'completed']
    default: 'pending'
    lowercase: true,
    required: true,
  }
```